x-pack/plugins/ingest_pipelines/public/index.ts

```diff
@@ -13,3 +13,5 @@ export function plugin() {
 
 export { INGEST_PIPELINES_APP_LOCATOR, INGEST_PIPELINES_PAGES } from './locator';
 export type { IngestPipelinesListParams } from './locator';
+export { apiService } from './application/services/api';
```

Comment:
I don't think we want to add a direct dependency on another plugin's API like this.

If we need to fetch pipelines in serverless search we'll need to implement it there.

Reply:
Thank you. I will update this part.

Reply:
Updated